lib/timeline/component/Group.js

```diff
@@ -486,7 +484,7 @@ Group.prototype._updateSubgroupsSizes = function () {
   if (me.subgroups) {
     for (var subgroup in me.subgroups) {
       var newStart = me.subgroups[subgroup].items[0].data.start;
-      var newEnd = me.subgroups[subgroup].items[0].data.end;
+      var newEnd = me.subgroups[subgroup].items[0].data.end - 1;
 
       me.subgroups[subgroup].items.forEach(function(item) {
         if (new Date(item.data.start) < new Date(newStart)) { 
```

Comment:
I understand this is not part of your change, but why is it necessary to convert to `Date` here? You can compare millisecond values directly. Is there perhaps a mixup of types for the start and end fields? That they can be either milliseconds or of type `Date`?

I hereby note that all comparisons between `Date` and milliseconds work, except the obvious:

```js
a = 1000000;
b = new Date(1000000);
a == b;    // false!
a === b;   // false!

// But following as expected
a < b;    // false
a <= b;   // true
a >= b;   // true
a > b;    // false
```

Which is just one more inconsistency in javascript. But it does mean that all `new Date()` parts can be removed in the conditions here. Just an observation.